src/routes/v1/oauth/start/start.js

```diff
@@ -0,0 +1,18 @@
+import { getClient } from "../../../../lib/oauth/config";
+import { generators } from "openid-client";
+
+export default async function(req, res) {
+  const providerName = req.query.provider;
+  const client = await getClient(providerName);
+  const nonce = generators.nonce();
+  const url = await client.authUrl(nonce);
+  const isDev = process.env.NODE_ENV === "development";
+  let options;
+  if (isDev) {
+    options = { httpOnly: true };
+  } else {
+    options = { httpOnly: true, secure: true };
+  }
```

Comment:
```suggestion
const options = { httpOnly: true };
if (!isDev) options.secure = true;
```

Nitpicking as usual but just something that could shorten it up here a little.